src/casp/_settings.py

```diff
@@ -0,0 +1,73 @@
+import json
```

Comment:
Sorry guys for not doing this in a separate PR. I decided to factor out settings to the root of the project.